mico-core/src/main/java/io/github/ust/mico/core/web/ServiceController.java

```diff
@@ -303,8 +310,8 @@
         MicoService savedService = serviceRepository.save(service);
 
         return ResponseEntity
-                .created(linkTo(methodOn(ServiceController.class).getServiceById(savedService.getId())).toUri())
-                .body(new Resource<>(savedService, getServiceLinks(savedService)));
+            .created(linkTo(methodOn(ServiceController.class).getServiceById(savedService.getId())).toUri())
+            .body(new Resource<>(savedService, getServiceLinks(savedService)));
     }
 
     @GetMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}" + "/dependers")
```

Comment:
Consider extracting path part `dependers` to a constant.

Reply:
Added a commit to #514 (PR for issue "Refactor Service Controller")